pkg/api/admin/openshiftcluster_example.go

```diff
@@ -0,0 +1,59 @@
+package admin
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+func exampleOpenShiftCluster() *OpenShiftCluster {
```

Comment:
I keep wondering if we should define this in pkg/api and convert it out to a versioned API in the places that it's needed

Reply:
+1 on this. And I think this should be on the Document Level and if you need `OpenShiftCluster` you just cherry-pick substructure. But I think this is separate issue/PR

Reply:
Can we move this to something like `/test/api` as we will need to re-use this all around the place

Reply:
Exampels are used in both: tests and swagger example generation. So it's a bit more complicated than just moving things around.

Anyway - I would prefer to deal with it separately from this PR.

Reply:
ok - please open a follow-up